WORKSPACE

```diff
@@ -60,11 +66,16 @@ stack_snapshot(
         "ansi-terminal",
         "async",
         "attoparsec",
+        "base",
```

Comment:
We have to add these packages (which we were getting from the GHC bindist before) so that `stack_snapshot` knows how to build our `vendored_packages`.